src/SwedbankPay.Sdk/PaymentInstruments/Invoice/PaymentRequestObject.cs

```diff
@@ -0,0 +1,61 @@
+﻿using SwedbankPay.Sdk.Common;
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+
+namespace SwedbankPay.Sdk.PaymentInstruments.Invoice
+{
+    public class PaymentRequestObject : IPaymentRequestObject
+    {
+        public PaymentRequestObject(Operation operation,
+                                                    PaymentIntent intent,
+                                                    CurrencyCode currency,
+                                                    List<IPrice> prices,
+                                                    string description,
+                                                    string payerReference,
+                                                    bool generatePaymentToken,
+                                                    bool generateReccurenceToken,
+                                                    string userAgent,
+                                                    CultureInfo language,
+                                                    IUrls urls,
+                                                    PayeeInfo payeeInfo,
+                                                    Dictionary<string, object> metadata = null,

```

Comment:
Codacy found an issue: [Use the overloading mechanism instead of the optional parameters.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-dotnet/pullRequest?prid=6290774)